nextcord/client.py

```diff
@@ -2181,29 +2195,48 @@ def get_application_command(self, command_id: int) -> Optional[BaseApplicationCo
         return self._connection.get_application_command(command_id)
 
     def get_application_command_from_signature(
-        self, name: str, cmd_type: Union[int, ApplicationCommandType], guild_id: Optional[int]
-    ) -> Optional[BaseApplicationCommand]:
+        self,
+        qualified_name: str,
+        *,
+        type: Union[int, ApplicationCommandType] = ApplicationCommandType.chat_input,
+        guild: Optional[Snowflake] = None,
```

Comment:
Why was this changed to accept guild objects? I don't see a good reason for it, considering the underlying method in the state still accepts the guild id anyways.

Reply:
I think changing it to be a union of `int` and `Snowflake` would be a good idea.